classes/Category.php

```diff
@@ -2441,4 +2441,15 @@ public function existsInShop($idShop)
 		WHERE `id_category` = ' . (int) $this->id . '
 		AND `id_shop` = ' . (int) $idShop, false);
     }
+
+    /**
+     * Indicates whether a category is ROOT for the shop.
+     * The root category is the one with no parent. It's a virtual category.
+     *
+     * @return bool
+     */
+    public function isRootCategory()
+    {
+        return 0 === (int) $this->id_parent || null === $this->id_parent;
```

Comment:
```suggestion
        return 0 === (int) $this->id_parent;
```
null cast in int equals 0